### DISCUSSION TITLE
Paxcounter in wilderness areas

### DISCUSSION THREAD

**Discussion by cyberman54** [cyberman54/ESP32-Paxcounter#959] [Show and tell]
I open this discussion thread, because i received several mails from users who deploy paxcounters in wilderness areas to get data on (in certain cases unwanted) footfall in those areas.

Users who have experience with paxcounter deployments in the wilderness are invited to share their knowledge, approaches and code here.

/cc @Qmick01 @TomRottler

**Comment by Qmick01** [comment]
Happy to share what I know.

I have deployed small numbers of Paxcounter units here in the UK running on battery packs in fairly remote areas. Have done this occasionally with no LORA coverage so relying on SD but its usually with coverage.

If you have any questions, please give me a shout.

**Reply by TomRottler** [reply]
Hi @Qmick01 ;
thanks for sharing your experiences. We are also working on an optional solution without LORA. Just a quick question: How do you solve the energy consumption of the devices? How big are your battery packs?

Tom

**Reply by t-huyeng** [reply]
May I ask why you are not using LoRa? Is the area too remote? Can you explain the use case or is it a company secret?

**Comment by cyberman54** [comment]
Via @TomRottler :
https://oberlab.de/projekte/cp.html
https://www.euregio-sbm.eu/projekte/ba0300077-esbm-connecting-peaks/

**Comment by Qmick01** [comment]
I have worked mostly with a pack of 4x 27650 LIPO batteries (22Ah.) And
also a deployment using a 25Ah 12v Lipo running through an DC/DC converter.

The 22Ah pack was powering a LILIGO LORA OLED board with Lora and BLE
sensing only enabled and a 600 second sleep cycle.
This config would give about 35 days of life on a 22ah 1s lipo pack.

The 25Ah pack has no LORA and was several months in the field with very
little measureable power consumption.

Note that the LEDs are removed for all these boards and the OLED display
switched off.

**Comment by TomRottler** [comment]
Hi @Qmick01 ,
thanks for the details. With your big power packs I guess you can log the data some weeks. Do you replace your battery packs cyclic and catch the logged data?
In our first project we placed some units with a tiny solar modul and a single 18650 accu in the mountains. These devices send on the daytime the logged data every 5 min. These systems are running now since fully 2 years without any maintenance in the wildlife with extreme weather situations and a lot of snow in the winter time.
Our current version 2 project will have some different use cases and we developed a PCB with all necessary components like the battery loader, external RTC, connectors for sensors and external battery blocks...
![bild_von_ios](https://user-images.githubusercontent.com/23116988/235787863-7eff0175-6d6a-4105-9ebc-bbcdfd836095.jpg)
(first picture of the prototype v2)

**Comment by Qmick01** [comment]
That's a very nice bit of kit.

We did a bunch of solar units also but it was a much s8mpler affair.

The panel was a USB solar unit purchased on Aliexpress for about £30 and
then modified slightly so the power was drawn directly from the 6 18650
batteries. This way we always know the actual state of the batteries which
is one of the data payloads we send.

The TTGO board was mounted on the back of the solar panel inside a 3d
printed enclosure designed to attach to the panel mounting system.

These units have been very reliable and have been running over a year
trouble free and the batteries have rarely dropped below 4 volts.

**Comment by cyberman54** [comment]
Regarding battery state: It is not necessary to transmit battery voltage by payload, if your LORAWAN server supports getting battery status information on MAC level. The paxcounter code transmits this information. Look in the code [here](https://github.com/cyberman54/ESP32-Paxcounter/blob/master/src/power.cpp#L256). This gives instant battery information with every data package which is sent by the node, regardless if payload or network packet. Thus, saves power and airtime.

**Comment by Qmick01** [comment]
Thanks.

Right now we send two payloads, one with battery one with Pax.

How would we use that instead?

**Reply by cyberman54** [reply]
For the TheThingsNetworks, look [here](https://www.thethingsindustries.com/docs/reference/api/end_device/) and scroll down the doc to field `battery_percentage`

![image](https://user-images.githubusercontent.com/10289836/235975586-eb6b1575-ee16-44fc-aa8c-09db815ec0c0.png)


For Loriot, look [here](https://docs.loriot.io/display/LNS/Device+status+DevStatusReq)

![image](https://user-images.githubusercontent.com/10289836/235975425-6e83abb9-1028-4341-b2a1-6ccb4922afa9.png)

**Reply by Qmick01** [reply]
I have just looked at this - but I honestly don't understand it. Has anyone tried to use it? How is it used?

**Comment by Qmick01** [comment]
In our case the landscape was too rugged and remote to get a gateway to
cover the units. And if we installed a gateway it would need power and
connectivity, all of which are tricky to provision in a remote landscape.
